hieradata_aws/class/production/account_api_db_admin.yaml

```diff
@@ -0,0 +1,13 @@
+govuk_env_sync::tasks:
```

Comment:
Should we comment this line out too? I'm not too sure how YAML interprets this when everything is commented - I'd guess it doesn't set to an object but instead sets it to null?

Reply:
Fair point. I'd assume it would just be treated as an empty array, but it could be interpreted as a null or an empty string. Probably safer to keep the whole thing commented out.